lesson10.md

```diff
@@ -24,9 +24,11 @@ function handleButtonClick() {
 
 ---
 
-### Recap: DOM object
+### Recap - In other words: DOM object
```

Comment:
I am not so sure about this, in other words?

Reply:
we don't say **DOM** especifically anywhere before and we should either say it's a new thing, or say we were already using it before, but not using its official name.
That's why I've added "in other words", but we could simply remove "Recap":
```
### DOM object
```

Reply:
I've just checked lesson 9, and I think it's ok to leave with "Recap" as well as the newly added little description :)